news/2021-12-02-skinning-contest-tides-of-winter-accouncement.md

```diff
@@ -0,0 +1,89 @@
+
+## Changes
+
+As per the feedback we've received from you, we've decided to make a couple of adjustments and improvements to this edition of the contest:
+
+- Each contestant can only participate with their own work, meaning that **collaborations/teams are not allowed**. Of course, you are permitted to ask others for feedback, but that should not go beyond exchanging ideas.
+- The file size of your skin as a whole must be **under 100MB**. As much as we like high-fidelity visual and aural content, we have to look out for those who have a poor system or internet connection.
+- At least **two** gamemodes must be skinned. In addition to this rule, we expect you to skin the **Mode Selection** panel as well.
+
+All of the changes above can be viewed in detail in the [Rules](#rules) section below. Make sure to have a read!
+
+## Participation
+
+To enter the competition, you must create a brand-new skin from scratch, ready to compete against skin creators from all over the world!
+
+In order for your submission to be valid, and also for everyone else to fall in love with your creation, you need to create a forum thread accompanying the skin. However you advertise your skin, though, is completely up to you. Said forum thread must be located under the [Skinning WIP subforum](https://osu.ppy.sh/community/forums/119).
+
+Once you have prepared your masterpiece, all you have to do is to leave a post under the [submission thread](https://osu.ppy.sh/community/forums/topics/1472846)!
+
+Since all entries will have their respective forum threads publicly available, contestants and their submissions will not be anonymized. In fact, we encourage you to show off your work as much as you wish!
+
+## Voting
+
+After the submission phase is over, the entries will be made available for community voting.
+
+The results of the community voting will determine the winner of the contest. Entry with the most votes wins!
+
+## Schedule
+
+- Submission phase: **4** weeks starting from the release of this newspost
+- Voting phase: **2** weeks starting from a separate newspost
+
+## Rules
+
+- Submissions must not contain inappropriate, malicious, or epileptic content, and should adhere to the [osu! community rules](/wiki/Rules).
+- All assets need to be created by yourself, or used with permission and proper credit to the authors. This includes, but is not limited to:
+  - fonts
+  - icons
+  - textures
+  - artworks (e.g. stock images or anime artwork)
+  - sounds
+- The majority of the in-game menu interface and at least two game modes have to be skinned. In other words, in addition to the gameplay elements, the following segments shouldn't be left default:
+  - Ranking Panel
+  - Song Selection
+  - Mode Selection
+  - Mod Icons
+  - Pause and Fail Menu
+  - Main Menu is optional, but highly encouraged
```

Comment:
```suggestion
  - Main Menu is optional, but highly encouraged.
```

Reply:
This isn't a full sentence, just a bullet point, so no . should be needed

Reply:
It *is* a full sentence, but I guess it can stay as is.